example/lib/main.dart

```diff
@@ -32,15 +42,27 @@ class _MyAppState extends State<MyApp> {
           centerTitle: true,
         ),
         body: SafeArea(
-          child: Stack(
-            children: [
-              AgoraVideoViewer(
-                client: client,
-              ),
-              AgoraVideoButtons(
-                client: client,
-              ),
-            ],
+          child: FutureBuilder(
+            future: futureInitialize,
+            builder: (context, snapshot) {
+              if (snapshot.connectionState == ConnectionState.waiting) {
+                return Center(child: CircularProgressIndicator());
+              }
+              if (snapshot.hasError) {
+                return Center(child: Text('Some error occurred.'));
+              }
```

Comment:
Could this be more descriptive?

Reply:
Sure.

Reply:
I'm thinking on "Coulnd't initialize AgoraClient", that is not something a user from an app would expect but as a demo of the lib I see no problem.

Reply:
Very true, it's just the example project, not the actual library.

Reply:
Then it's done :smiley: